features/app/components/native/App.js

```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
+import { Provider } from 'react-redux';
+import { Navigator } from 'react-native';
+import { init, WelcomePage } from '../../../welcome';
+import { Conference } from '../../../conference';
```

Comment:
Our coding style requires that imports are sorted by import location in alphabetical order:

../../../conference
../../../welcome